_docs/_latest/setup/upgrade.md

```diff
@@ -398,6 +398,45 @@ Example command: `gcloud compute instances reset forseti-server-vm-70ce82f --zon
 1. Configuration file `forseti_conf_server.yaml` updates:  
 Forseti is updated to be usable on a non organization resource.
 
+{% capture upgrading_2_4_0_to_2_5_0 %}
```

Comment:
We also need to mention that that Cloud Asset API needs to be enabled.

Reply:
Done.